Modules/vc-module-notifications/VirtoCommerce.NotificationsModule.LiquidRenderer/Filters/TranslationFilter.cs

```diff
@@ -11,10 +11,17 @@ public static string Translate(TemplateContext context, string key, string langu
             var result = key;
 
             var localizationService = (ITranslationService)context.GetValue(new ScriptVariableGlobal(nameof(NotificationScriptObject.TranslationService)));
+
             if (string.IsNullOrEmpty(language))
             {
                 language = context.GetValue(new ScriptVariableGlobal(nameof(NotificationScriptObject.Language)))?.ToString();
             }
+
+            if (!string.IsNullOrEmpty(language))
```

Comment:
Why do we need this a key transformation here? We don't use language as par of key in our translation tables

Reply:
no need. you right!